### PR TITLE
Release v1.36.3

### PR DESCRIPTION
**Pull request by tanmay-db**
### New Features and Improvements
 * Explicitly set securable field when reading [databricks_grants](https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/grants) or [databricks_grant](https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/grant) ([#3246](https://github.com/databricks/terraform-provider-databricks/pull/3246)).

### Documentation Changes
 * Added information on `id` and other exposed attributes where appropriate ([#3237](https://github.com/databricks/terraform-provider-databricks/pull/3237)).
 * Fixed docs in metastore databricks_grants example ([#3239](https://github.com/databricks/terraform-provider-databricks/pull/3239)).

### Exporter
 * Detect & handle deleted workspace objects (notebooks/files/directories) when running in incremental mode ([#3225](https://github.com/databricks/terraform-provider-databricks/pull/3225)).

### Internal Changes
 * Make IterFields take in aliases ([#3207](https://github.com/databricks/terraform-provider-databricks/pull/3207)).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # Version changelog
 
+## 1.36.3
+
+### New Features and Improvements
+ * Explicitly set securable field when reading [databricks_grants](https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/grants) or [databricks_grant](https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/grant) ([#3246](https://github.com/databricks/terraform-provider-databricks/pull/3246)).
+
+### Documentation Changes
+ * Added information on `id` and other exposed attributes where appropriate ([#3237](https://github.com/databricks/terraform-provider-databricks/pull/3237)).
+ * Fixed docs in metastore databricks_grants example ([#3239](https://github.com/databricks/terraform-provider-databricks/pull/3239)).
```
Comment (alexott) [inline CHANGELOG.md:10]: can we add backticks around `databricks_grants` ?

**Review by alexott — APPROVED** [review]
small nit

**Comment by tanmay-db** [comment]
waiting for nightly to finish

**Comment by tanmay-db** [comment]
2 tests failed due to client timeout, they look unrelated to changes after release. Rerunning them again.

**Comment by tanmay-db** [comment]
All passed on retry. Going ahead with release.

**Pull request merged**